app/models/user.rb

```diff
@@ -58,8 +58,8 @@ def as_json(_options = {})
       admin: admin }
     if (_options[:follows])
       json['follows'] = {
-        topics: following.where(followed_type: 'Topic').to_a.map(&:followed_id),
-        maps: following.where(followed_type: 'Map').to_a.map(&:followed_id)
+        topics: following.where(muted: false, followed_type: 'Topic').to_a.map(&:followed_id),
```

Comment:
These just treat muted follows as if they don't exist. There's one tiny caveat here, which is that the front end doesn't know in which circumstances to avoid flipping a non-follow to a follow, in the case of an edit, but I am ok with that tiny gap, which will be resolved when they refresh the page